### ISSUE TITLE
mod-socket-io JavaScript

### ISSUE THREAD

**Issue by yjgaia** [keesun/mod-socket-io#22]
안녕하세요.
Vert.x에서는 JavaScript 또한 지원하는데, mod-socket-io를 JavaScript 환경에서 사용할 수 있는 방법이 있을까요?

**Comment by keesun** [comment]
네 저도 아직 자바로 만든 모듈 위에 자바스크립트 API를 얹는 방법을 모르겠네요. 흠... 이 이슈는 아래에 좀 더 급해 보이는 이슈 먼저 처리되면 진행할께요.

감사합니다.

**Comment by yjgaia** [comment]
감사합니다. :)
